csrc/multidevice/pipeline.cpp

```diff
@@ -152,9 +152,6 @@ class PipelineBuilder final {
         // then add the newly created PipelineVal as an input of the pipeline
         if (isGlobalInput(val)) {
           pipeline_->addInput(p_val);
-          NVF_ERROR(
```

Comment:
Why is this removed?

Reply:
we don't need this restriction. We want the inputs to be possibly sharded